al2/aarch64/standard/1.0/runtimes.yml

```diff
@@ -0,0 +1,126 @@
+version: 0.1
+
+runtimes:
+  java:
+    versions:
+      corretto11:
+        commands:
+          - echo "Installing corretto (OpenJDK) version 11 ..."
+
+          - export JAVA_HOME="$JAVA_11_HOME"
+
+          - export JRE_HOME="$JRE_11_HOME"
+
+          - export JDK_HOME="$JDK_11_HOME"
+
+          - |-
+            for tool_path in "$JAVA_HOME"/bin/*; 
+             do tool=`basename "$tool_path"`;
+              if [ $tool != 'java-rmi.cgi' ]; 
+              then 
+               rm -f /usr/bin/$tool /var/lib/alternatives/$tool \
+                && update-alternatives --install /usr/bin/$tool $tool $tool_path 20000;
+              fi; 
+            done
+      corretto8:
+        commands:
+          - echo "Installing corretto (OpenJDK) version 8 ..."
+
+          - export JAVA_HOME="$JAVA_8_HOME"
+
+          - export JRE_HOME="$JRE_8_HOME"
+
+          - export JDK_HOME="$JDK_8_HOME"
+
+          - |-
+            for tool_path in "$JAVA_8_HOME"/bin/* "$JRE_8_HOME"/bin/*;
+             do tool=`basename "$tool_path"`;
+              if [ $tool != 'java-rmi.cgi' ];
+              then
+               rm -f /usr/bin/$tool /var/lib/alternatives/$tool \
+                && update-alternatives --install /usr/bin/$tool $tool $tool_path 20000;
+              fi;
+            done
+  golang:
+    versions:
+      1.12:
+        commands:
+          - echo "Installing Go version 1.12 ..."
+          - rm -rf /usr/local/go
+          - ln -s /usr/local/go12 /usr/local/go
+      1.13:
```

Comment:
should we remove this?

Reply:
Go 1.13 is default go runtime on this image. This echo message is just a notification on the build logs. This is an expected behavior.